products/template/sql/product/_product_models.yml

```diff
@@ -13,24 +13,11 @@ models:
       - name: wkb_geometry
         tests:
           - not_null:
-              where: "bbl is not null"
+              where: bbl is not null
```

Comment:
what does "where" tag do?

Reply:
it limits a test to records where the condition is true ([docs](https://docs.getdbt.com/reference/resource-configs/where))

Reply:
niceeee